packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -2937,11 +2937,31 @@ void main() {
         );
 
         expect(getDecoratorRect(tester).size, const Size(800.0, 56.0));
+        // Label line height is forced to 1.0 and font size is 16.0,
+        // the label should be vertically centered (20 pixels above and below).
         expect(getLabelRect(tester).top, 20.0);
         expect(getLabelRect(tester).bottom, 36.0);
       },
     );
 
+    testWidgets('Ambient theme floatingLabelBehavior is used', (WidgetTester tester) async {
```

Comment:
Maybe comment that this is a regression test for https://github.com/flutter/flutter/issues/71813?

Reply:
> Your PR description says this adds 8 tests but I only see 1

Sorry for this, the description was wrong (copy/paste from another PR filed on the same day 😅 ).